README.rst

```diff
@@ -1,19 +1,21 @@
 GeoPySpark
-***********
+**********
+
 .. image:: https://travis-ci.org/locationtech-labs/geopyspark.svg?branch=master
    :target: https://travis-ci.org/locationtech-labs/geopyspark
 
 .. image:: https://readthedocs.org/projects/geopyspark/badge/?version=latest
    :target: https://geopyspark.readthedocs.io/en/latest/?badge=latest
 
+Documentation: https://geopyspark.readthedocs.io
```

Comment:
Do we need a link to the documentation right below the badge that links to the documentation?

Reply:
Only anecdotal, but Jenny, Ross and I were all unaware that there was a readthedocs page. I think I knew that the docs badge was also a link, but my brain skipped over the fact that I could click that. Given that the documentation is useful, I personally still think a direct link is worth the space as a callout. Happy to revert though.

Reply:
Ah, I see. In that case I'm okay with making the link explicit then. In that case I think we should make an explicit Gitter link as well since the badge alone might not be enough.